.github/workflows/django.yml

```diff
@@ -34,6 +34,9 @@ jobs:
           - python-version: "3.11"
             django-version: Django==3.2
```

Comment:
In some libs, you are dropping support for Django 3.x. As this version is no longer supported, i guess we can drop support too.